src/converter/dimacs/DimacsLogExp.spec.ts

```diff
@@ -0,0 +1,97 @@
+import each from "jest-each";
```

Comment:
by jests convention, test should be located in the `__tests__` folder at the root level of the project

-> https://nextjs.org/docs/testing#creating-your-tests